.github/workflows/intel.yml

```diff
@@ -124,27 +116,27 @@ jobs:
         key: ccache-${{ github.workflow }}-${{ github.job }}-git-${{ github.sha }}
         restore-keys: |
              ccache-${{ github.workflow }}-${{ github.job }}-git-
-    - name: build
+    - name: Build & Install
       env: {CXXFLAGS: "-Werror"}
       run: |
         export CCACHE_COMPRESS=1
         export CCACHE_COMPRESSLEVEL=10
         export CCACHE_MAXSIZE=40M
+        export CCACHE_DEPEND=1
```

Comment:
Usually, CCACHE_DEPEND=1 should not be used. We use it in the sycl builds as a workaround because otherwise `icpx -fsycl` does not work with ccache.

Reply:
Done.